### PR TITLE
merged gateway content with the base project

### PR DESCRIPTION
**Pull request by HEskandari**
Merges the gateway and the base into one project

### CODE REVIEW

**Comment by bording** [comment]
Given the conflicts that are going to happen between this and #2, I think we should go ahead and merge #2 first, and then make these changes.

File: src/ViewModelComposition.sln [hunk 1]
```diff
@@ -1,37 +1,31 @@
...
+		{360B1F7B-A2A0-4C44-869B-67E9DCBD2531}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{360B1F7B-A2A0-4C44-869B-67E9DCBD2531}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {6575C8B8-C232-4117-BA32-79DFFA5497C4}
+	EndGlobalSection
+EndGlobal
```
Comment (adamralph) [inline src/ViewModelComposition.sln]: This is a line ending problem. The line endings have been changed from LF to CRLF.

Comment (bording) [reply src/ViewModelComposition.sln]: This has been fixed already in #2, which adds even more to my recommendation of merging #2 first, and then redoing this PR.

Comment (adamralph) [reply src/ViewModelComposition.sln]: 👍 to merging #2 first

**Comment by HEskandari** [comment]
Closing this, opening another PR.

**Pull request closed without merging**